Comparing source compatibility of /home/someone/git/com.github/io7m/jtensors/com.io7m.jtensors.core/target/com.io7m.jtensors.core-8.1.0.jar against /home/someone/.m2/repository/com/io7m/jtensors/com.io7m.jtensors.core/8.0.0/com.io7m.jtensors.core-8.0.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors2D  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.parameterized.vectors.PVector2D, com.io7m.jtensors.core.parameterized.vectors.PVector2D)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors2F  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.parameterized.vectors.PVector2F, com.io7m.jtensors.core.parameterized.vectors.PVector2F)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors2I  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.parameterized.vectors.PVector2I, com.io7m.jtensors.core.parameterized.vectors.PVector2I)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors2L  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.parameterized.vectors.PVector2L, com.io7m.jtensors.core.parameterized.vectors.PVector2L)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors3D  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.parameterized.vectors.PVector3D, com.io7m.jtensors.core.parameterized.vectors.PVector3D)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors3F  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.parameterized.vectors.PVector3F, com.io7m.jtensors.core.parameterized.vectors.PVector3F)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors3I  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.parameterized.vectors.PVector3I, com.io7m.jtensors.core.parameterized.vectors.PVector3I)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors3L  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.parameterized.vectors.PVector3L, com.io7m.jtensors.core.parameterized.vectors.PVector3L)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors4D  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.parameterized.vectors.PVector4D, com.io7m.jtensors.core.parameterized.vectors.PVector4D)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors4F  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.parameterized.vectors.PVector4F, com.io7m.jtensors.core.parameterized.vectors.PVector4F)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors4I  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.parameterized.vectors.PVector4I, com.io7m.jtensors.core.parameterized.vectors.PVector4I)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.parameterized.vectors.PVectors4L  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.parameterized.vectors.PVector4L, com.io7m.jtensors.core.parameterized.vectors.PVector4L)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors2D  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.unparameterized.vectors.Vector2D, com.io7m.jtensors.core.unparameterized.vectors.Vector2D)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors2F  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.unparameterized.vectors.Vector2F, com.io7m.jtensors.core.unparameterized.vectors.Vector2F)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors2I  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.unparameterized.vectors.Vector2I, com.io7m.jtensors.core.unparameterized.vectors.Vector2I)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors2L  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.unparameterized.vectors.Vector2L, com.io7m.jtensors.core.unparameterized.vectors.Vector2L)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors3D  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.unparameterized.vectors.Vector3D, com.io7m.jtensors.core.unparameterized.vectors.Vector3D)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors3F  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.unparameterized.vectors.Vector3F, com.io7m.jtensors.core.unparameterized.vectors.Vector3F)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors3I  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.unparameterized.vectors.Vector3I, com.io7m.jtensors.core.unparameterized.vectors.Vector3I)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors3L  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.unparameterized.vectors.Vector3L, com.io7m.jtensors.core.unparameterized.vectors.Vector3L)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors4D  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.unparameterized.vectors.Vector4D, com.io7m.jtensors.core.unparameterized.vectors.Vector4D)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors4F  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) double distance(com.io7m.jtensors.core.unparameterized.vectors.Vector4F, com.io7m.jtensors.core.unparameterized.vectors.Vector4F)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors4I  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.unparameterized.vectors.Vector4I, com.io7m.jtensors.core.unparameterized.vectors.Vector4I)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jtensors.core.unparameterized.vectors.Vectors4L  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) long distance(com.io7m.jtensors.core.unparameterized.vectors.Vector4L, com.io7m.jtensors.core.unparameterized.vectors.Vector4L)
